src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -3230,6 +3230,10 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 "FileScanner_PasswordError_title" = "Falsches Passwort";
 
+"FileScanner_AlreadyRegistered_title" = "Signatur ungültig";
```

Comment:
The label of the error message should be: "Zertifikat schon registriert"

Reply:
I changed it. Please review again @KathrinLuecke